editor/gui/editor_file_dialog.cpp

```diff
@@ -967,27 +982,44 @@ void EditorFileDialog::update_file_list() {
 			continue;
 		}
 
-		if (show_hidden_files) {
-			if (!dir_access->current_is_dir()) {
-				files.push_back(item);
-			} else {
-				dirs.push_back(item);
-			}
-		} else if (!dir_access->current_is_hidden()) {
-			String full_path = cdir == "res://" ? item : dir_access->get_current_dir() + "/" + item;
-			if (dir_access->current_is_dir()) {
-				if (Engine::get_singleton()->is_project_manager_hint() || !EditorFileSystem::_should_skip_directory(full_path)) {
-					dirs.push_back(item);
+		bool matches_search = true;
+		if (searched_string.length() > 0) {
+			matches_search = item.to_lower().find(searched_string) != -1;
+		}
+
+		FileInfo fi;
```

Comment:
```suggestion
		FileInfo file_info;
```
A bit nitpicky, but for consistency :)